@boltex I pulled this branch to investigate the pylint failures.

### flake8 errors

```console
flake8_leo.py
c:\Repos\leo-editor\leo\core\leoserver.py:5718:9: F824 `global wsCert` is unused: name is never assigned in scope
        global wsHost, wsPort, wsLimit, wsPersist, wsSkipDirty, argFile, wsPassword, wsCert, wsKey
        ^
c:\Repos\leo-editor\leo\core\leoserver.py:5718:9: F824 `global wsKey` is unused: name is never assigned in scope
        global wsHost, wsPort, wsLimit, wsPersist, wsSkipDirty, argFile, wsPassword, wsCert, wsKey
        ^
c:\Repos\leo-editor\leo\core\leoserver.py:5884:19: F541 f-string is missing placeholders
            print(f"Error: Certificate files not found. Falling back to ws://")
```

### ruff errors

```console
ruff_leo.py
F541 [*] f-string without any placeholders
    --> leo\core\leoserver.py:5884:19
     |
5882 |         # Verify files actually exist on disk
5883 |         if not os.path.exists(cert_path) or not os.path.exists(key_path):
5884 |             print(f"Error: Certificate files not found. Falling back to ws://")
     |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
5885 |             return None
     |
help: Remove extraneous `f` prefix
```

### Reformatted files

Rev 227e5be reformats both files per Leo's beautifier settings.

--- a/leo/core/leoserver.py
+++ b/leo/core/leoserver.py
@@ -5844,6 +5844,8 @@ def main() -> None:  # pragma: no cover (tested in client)
         wsPersist = bool(args.wsPersist)
         wsSkipDirty = bool(args.wsSkipDirty)
         wsPassword = args.wsPassword
+        wsCert = args.wsCert
+        wsKey = args.wsKey
         argFile = args.argFile
         if args.traces:
             ok = True
@@ -5881,7 +5883,7 @@ def main() -> None:  # pragma: no cover (tested in client)
 
         # Verify files actually exist on disk
         if not os.path.exists(cert_path) or not os.path.exists(key_path):
-            print(f"Error: Certificate files not found. Falling back to ws://")
+            print("Error: Certificate files not found. Falling back to ws://")
             return None
 
         print(f"Running in secure mode (wss://) using {cert_path}")

